src/auro-input.js

```diff
@@ -67,9 +67,9 @@ export default class AuroInput extends BaseInput {
         aria-invalid="${!this.isValid}"
         placeholder=${this.getPlaceholder()}
         lang="${ifDefined(this.lang)}"
-        spellcheck="${ifDefined(this.type === 'password' ? `false` : undefined)}"
-        autocorrect="${ifDefined(this.type === 'password' ? `off` : undefined)}"
-        autocapitalize="${ifDefined(this.type === 'password' ? `none` : undefined)}"
+        spellcheck="${ifDefined(this.spellcheck ? this.spellcheck : this.setInputDefinition())}"
+        autocorrect="${ifDefined(this.autocorrect ? this.autocorrect : this.setInputDefinition())}"
+        autocapitalize="${ifDefined(this.autocapitalize ? this.autocapitalize : this.setInputDefinition())}"
```

Comment:
I don't think we should be calling this method multiple times, since it mutates the element's properties and doesn't return anything.